Mathlib/RingTheory/ZMod/Torsion.lean

```diff
@@ -0,0 +1,29 @@
+/-
+Copyright (c) 2025 Yakov Pechersky. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yakov Pechersky
+-/
+import Mathlib.RingTheory.RootsOfUnity.EnoughRootsOfUnity
+import Mathlib.FieldTheory.Finite.Basic
+
+/-!
+# Torsion group of `ZMod p` for prime `p`
+
+This file shows that the `ZMod p` has `p - 1` roots-of-unity.
+
+-/
+
+lemma ZMod.rootsOfUnity_eq_top {p : ℕ} [Fact p.Prime] :
```

Comment:
It's only one declaration (and an instance), but I think it makes sense to `namespace ZMod`.